examples/multi_script_CLI.py

```diff
@@ -1,97 +1,607 @@
+            print("we will skip media that have lesser following from this value (enter to use default number: 10)")
+            f.write(str(int(sys.stdin.readline().strip()or "10")) + "\n")
+            print("Maximal followers to following_ratio (enter to use default number: 10)")
+            f.write(str(int(sys.stdin.readline().strip()or "10")) + "\n")
+            print("Maximal following to followers_ratio (enter to use default number: 2)")
+            f.write(str(int(sys.stdin.readline().strip()or "2")) + "\n")
+            print("Minimal media the account you will follow have.")
+            print("we will skip media that have lesser media from this value (enter to use default number: 7)")
+            f.write(str(int(sys.stdin.readline().strip()or "7")) + "\n")
+            print("Delay from one like to another like you will perform (enter to use default number: 30)")
+            f.write(str(int(sys.stdin.readline().strip()or "30")) + "\n")
+            print("Delay from one unlike to another unlike you will perform (enter to use default number: 10)")
+            f.write(str(int(sys.stdin.readline().strip()or "10")) + "\n")
+            print("Delay from one follow to another follow you will perform (enter to use default number: 30)")
+            f.write(str(int(sys.stdin.readline().strip()or "30")) + "\n")
+            print("Delay from one unfollow to another unfollow you will perform (enter to use default number: 30)")
+            f.write(str(int(sys.stdin.readline().strip()or "30")) + "\n")
+            print("Delay from one comment to another comment you will perform (enter to use default number: 60)")
+            f.write(str(int(sys.stdin.readline().strip()or "60")) + "\n")
+            print("Want to use proxy? insert your proxy or leave it blank if no. (just enter)")
+            f.write(str(str(sys.stdin.readline().strip()or "None")) + "\n")
+            print("done with all settings")
+            break
 
+def parameter_setting():
+    print("current parameter\n")
+    f=open(setting)
+    data=f.readlines()
+    print("Max likes per day: "+data[0])
+    print("Max unlikes per day: "+data[1])
+    print("Max follows per day: "+data[2])
+    print("Max unfollows per day: "+data[3])
+    print("Max comments per day: "+data[4])
+    print("Max likes to like: "+data[5])
+    print("Max followers to follow: "+data[6])
+    print("Min followers to follow: "+data[7])
+    print("Max following to follow: "+data[8])
+    print("Min following to follow: "+data[9])
+    print("Max followers to following_ratio: "+data[10])
+    print("Max following to followers_ratio: "+data[11])
+    print("Min media_count to follow:"+data[12])
+    print("Like delay: "+data[13])
+    print("Unlike delay: "+data[14])
+    print("Follow delay: "+data[15])
+    print("Unfollow delay: "+data[16])
+    print("Comment delay: "+data[17])
+    print("Proxy: "+data[18])
 
+def username_adder():
```

Comment:
doubling the code - make one function with arguments (file_name) instead of several equal.